modules/webxr/native/library_godot_webxr.js

```diff
@@ -557,12 +593,34 @@ const GodotWebXR = {
 			return 0;
 		}
 
-		const axes_count = controller.gamepad.axes.length;
+		let axes = controller.gamepad.axes;
+		if (controller.gamepad.mapping === 'xr-standard' && p_xr_standard_mapping) {
+			const trigger_axis = (controller.gamepad.buttons[0].value * 2.0) - 1.0;
+			const grip_axis = (controller.gamepad.buttons[1].value * 2.0) - 1.0;
```

Comment:
@BastiaanOlij Are the trigger and grip axes meant to be from `-1.0` to `1.0` like normal axes? Or, `0.0` to `1.0`? I made them like normal axes here, but I'm not sure what's expected.

Reply:
0.0 (unpressed) to 1.0 (pressed) :)

Reply:
Ok, thanks! I'll update the PR in a little bit